edit/style.go

```diff
@@ -7,19 +7,19 @@ var (
 	//styleForPrompt           = ""
 	//styleForRPrompt          = "inverse"
 	styleForCompleted        = styles{"dim"}
+	styleForCompletedHistory = styles{"dim"}
 	styleForMode             = styles{"bold", "lightgray", "magenta"}
 	styleForTip              = styles{}
-	styleForCompletedHistory = styles{"dim"}
 	styleForFilter           = styles{"underlined"}
 	styleForSelected         = styles{"inverse"}
 	styleForScrollBarArea    = styles{"magenta"}
 	styleForScrollBarThumb   = styles{"magenta", "inverse"}
 	styleForSideArrow        = styles{"inverse"}
 
 	// Use black text on white for completion listing.
-	styleForCompletion = styles{"black", "bg_white"}
+	styleForCompletion = styles{}
 	// Use white text on black for selected completion.
-	styleForSelectedCompletion = "inverse"
+	styleForSelectedCompletion = styles{"black", "bg-white", "bold"}
```

Comment:
This is assuming that the default is white-on-black, which may not be the case (e.g. macOS's terminal). Simply use `"inverse"` to agree with `styleForSelected`. I prefer to leave out `bold` as well to agree with it.

The decision to use black-on-white for completion and white-on-black for selected completion was said in #184, but it doesn't work well with ls colors, so it makes sense to revert it.